CrackingCodingInterview/vDimple/04 Trees and Graphs/GraphsAlgorithms/PriorityQueue_MaxHeap/PriorityQueue_MaxHeap.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#ifdef PRIORITYQUEUEMAXHEAP_EXPORTS
+#define PRIORITYQUEUEMAXHEAP_API __declspec(dllexport)
+#else
+#define PRIORITYQUEUEMAXHEAP_API __declspec(dllimport)
+#endif // PRIORITYQUEUEMAXHEAP_EXPORTS
+
+#include<iostream>
+#include<vector>
+
+class PRIORITYQUEUEMAXHEAP_API MaxHeap
```

Comment:
We could write generic class using templates.

Reply:
Incorporated review comment.